ballerina/build.gradle

```diff
@@ -45,6 +45,8 @@ def ballerinaLinuxDistributionZip = file("$project.buildDir/distributions/baller
 def ballerinaMacDistributionZip = file("$project.buildDir/distributions/ballerina-macos-${ballerinaVersion}.zip")
 def ballerinaWindowsDistributionZip = file("$project.buildDir/distributions/ballerina-windows-${ballerinaVersion}.zip")
 
+def installerVersion = new Random().nextInt(10 ** 10)
```

Comment:
Lets include this in gradle.properties file and update it during the `release` gradle task
So it generate only during release and get committed during that time